data/xp-pen-deco-pro-mw.tablet

```diff
@@ -20,7 +22,7 @@
 [Device]
 Name=Hanvon Ugee Technology Co.,Ltd Deco Pro MW
 ModelName=
-DeviceMatch=usb|28bd|0934
+DeviceMatch=usb|28bd|0904|11 inch PenTablet;usb|28bd|0904|11 inch PenTablet Keyboard;usb|28bd|0904|11 inch PenTablet Mouse;usb|28bd|0934|11 inch PenTablet;usb|28bd|0934|11 inch PenTablet Keyboard;usb|28bd|0934|11 inch PenTablet Mouse
```

Comment:
are those usb IDs not unique? because if they are unique we don't need the name matches - those are only for devices where the pid is shared.